opencti-platform/opencti-front/src/private/components/common/files/FileManager.jsx

```diff
@@ -161,7 +161,7 @@ const FileManager = ({
     if (isCsvConnector && configuration && objectMarking) {
       const parsedConfig = JSON.parse(configuration);
       if (typeof parsedConfig === 'object') {
-        parsedConfig.user_chosen_markings = objectMarking.map((marking) => marking.value);
+        parsedConfig.markings = objectMarking.map((marking) => marking.value);
```

Comment:
why change this property name here ? I think it should stay user_chosen_markings